metal/loadbalancers/metallb/configmap.go

```diff
@@ -484,51 +487,67 @@ func (p *Peer) EqualIgnoreService(o *Peer) bool {
 }
 
 // Services list of services that this peer supports
-func (p *Peer) Services() []string {
+func (p *Peer) Services() []Resource {
+	var services []Resource
 	for _, ns := range p.NodeSelectors {
+		var name, namespace string
 		for k, v := range ns.MatchLabels {
-			if k == serviceNameKey {
-				return strings.Split(v, ",")
+			switch k {
+			case serviceNameKey:
```

Comment:
What are your thoughts on _healing_ the old format?  We would detect when `serviceNameKey` contains the old format and append any/all services detected.

I presume we would need to build a similar capability into the other for loops walking nodeselectors (which makes me wonder if these could be more consolidated).

Reply:
Is there any point? Given that the old format is broken, and shouldn't work, why bother?

Reply:
Fair.